### PR TITLE
remove global object, rename tokens.json to global.json

### PR DESCRIPTION
**Pull request by CCAyl**

### CODE REVIEW

File: data/global.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "global": {
+  "tokens":{
```
Comment (CCAyl) [inline data/global.json:2]: can something ELI5 why we need this top level object? i'd love to just nix it all together but typography in particular gets mad

Comment (iryanclarke) [reply data/global.json:2]: I think its due to how Figma tokens is expecting to be formatted, so it can read it properly?

Comment (CCAyl) [reply data/global.json:2]: i thought this was something we added, when we thought we would be adding light theme within this file also 

image
